Gopkg.lock

```diff
@@ -2,67 +2,78 @@
 
 
 [[projects]]
+  digest = "1:9f3b30d9f8e0d7040f729b82dcbc8f0dead820a133b3147ce355fc451f32d761"
```

Comment:
There's lots of noise from `digest`+`pruneopts` being brought back - they were removed by a previous dependency update commit, I guess because of older <0.5.x dep was used; hopefully everyone has 0.5+ by now so next time there should be less noise; if we notice they are being removed in the future, we should check which dep version used, ensure latest one.